README.md

```diff
@@ -1,42 +1,75 @@
 [![Build Status](https://travis-ci.org/JSJitsu/ai-battle-website.svg?branch=master)](https://travis-ci.org/JSJitsu/ai-battle-website)
-# Javascript Battle
+# Javascript Fight Club
 
-Every day there is an epic battle between legendary heroes.  Control your hero with either a pre-defined function from our repo or your own submitted function (your hero’s “brain”), and see the results of each day’s battle as your hero’s prowess is tested against the competition.
+Tool to run the games that the website is able to replay. Real or test data can be used to generate a battle.
 
-Sign up is as easy as logging into your github account, forking a “starter” repository, and opting in to the daily battle.  Learn from your past mistakes and make your hero smarter by customizing your hero.js file, and feel the power as your hero dominates the battlefield.
+## Playing The Game
+Visit [https://jsfight.club](https://jsfight.club).
```

Comment:
This is probably fine but I'd maybe point out that the code for this is in this repo? I dunno probably doesn't matter.